packages/gatsby-theme/src/components/Loading/index.js

```diff
@@ -77,25 +83,29 @@ const Loading = ({ type }) => {
             }}
           />
 
-          <rect
+          <Box
+            as='rect'
             x='55%'
             rx='3'
             ry='3'
             sx={{ y: "60", width: ["0%", "40%"], height: [230, null, 340] }}
           />
-          <rect
+          <Box
+            as='rect'
             x='55%'
             rx='3'
             ry='3'
             sx={{ y: ["310", null, "420"], width: ["0%", "15%"], height: 20 }}
           />
-          <rect
+          <Box
+            as='rect'
             x='55%'
             rx='3'
             ry='3'
             sx={{ y: ["340", null, "450"], width: ["0%", "40%"], height: 20 }}
```

Comment:
Maybe we could use the __css property instead of the sx, what do you think?

Reply:
Sure, thank you! :)